bare-metal/container-linux/kubernetes/cl/controller.yaml.tmpl

```diff
@@ -81,7 +81,7 @@ systemd:
         ExecStartPre=/bin/mkdir -p /var/lib/kubelet/volumeplugins
         ExecStartPre=/usr/bin/bash -c "grep 'certificate-authority-data' /etc/kubernetes/kubeconfig | awk '{print $2}' | base64 -d > /etc/kubernetes/ca.crt"
         ExecStartPre=-/usr/bin/rkt rm --uuid-file=/var/cache/kubelet-pod.uuid
-        ExecStart=/usr/lib/coreos/kubelet-wrapper \
+        ExecStart=/usr/lib/${os_flavor}/kubelet-wrapper \
```

Comment:
This isn't needed for this change. Flatcar keeps the kubelet-wrapper in the same location.